pettingzoo/classic/connect_four/connect_four.py

```diff
@@ -2,11 +2,21 @@
 from gym import spaces
 import numpy as np
 import warnings
+import os
+
 
 from pettingzoo.utils import wrappers
 from pettingzoo.utils.agent_selector import agent_selector
 
 
+def get_image(path):
+    import pygame
+    from os import path as os_path
+    cwd = os_path.dirname(__file__)
+    image = pygame.image.load(cwd + '/' + path)
+    return image
+
+
 def env():
     env = raw_env()
     env = wrappers.CaptureStdoutWrapper(env)
```

Comment:
This should also be removed. This implements a `ansi` render mode, that will no longer work once the prints are removed.

Reply:
By this I mean the CaptureStdoutWrapper specifically.